packages/server/src/fhir/operations/expand.ts

```diff
@@ -410,7 +410,13 @@ export function expansionQuery(
 
 function addExpansionFilters(query: SelectQuery, params: ValueSetExpandParameters): SelectQuery {
   if (params.filter) {
-    query.where('display', '!=', null).where('display', 'TSVECTOR_ENGLISH', filterToTsvectorQuery(params.filter));
+    query
+      .whereExpr(
+        new Conjunction(params.filter.split(/\s+/g).map((filter) => new Condition('display', 'LIKE', `%${filter}%`)))
```

Comment:
What happens if `params.filter` includes a `%` ?

Are there any other special characters that need to be handled?